demo-transactions/src/main/resources/db/read/V0_0_1__account_balance_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE `cqrs-meets-hexagonal-read-db`.account_balance(
```

Comment:
Does it add any confusion to on the one side use the wording 'read' and 'write' (within db resources) and on the other side 'command' and 'query' (in package structures)?
Maybe we should stick to `command-model` and `query-model` instead of write and read (especially since on both sides data is writted and read).

Reply:
Yeah i changed to a better suited name, which means that you need to clear the database.